tsconfig.json

```diff
@@ -1,9 +1,15 @@
 {
   "compilerOptions": {
-    "outDir": "dist",    
+    "outDir": "dist",
     "baseUrl": ".",
     "paths": {
       "~/*": ["src/*"],
+      "@polymeshassociation/polymesh-sdk": [
+        "node_modules/@polymeshassociation/confidential-polymesh-sdk"
+      ],
+      "@polymeshassociation/polymesh-sdk/*": [
+        "node_modules/@polymeshassociation/confidential-polymesh-sdk/*"
```

Comment:
seems like jest isn't able to pick this up.

maybe we need to do find + replace across the code base instead? Might be annoying to have to update every import line when syncing with upstream though

Reply:
I need to added in jest as well